### PR TITLE
[12.0][ADD] stock_move_packaging_qty

### PR DESCRIPTION
**Pull request by MateuGForgeFlow**
Add the packaging fields in the stock moves.
@LoisRForgeFlow

### CODE REVIEW

File: stock_move_packaging_qty/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Stock Packaging Qty",
+    "summary": "Add packaging fields in the stock moves",
+    "version": "12.0.1.1.0",
```
Comment (LoisRForgeFlow) [inline stock_move_packaging_qty/__manifest__.py]: 12.0.1.0.0

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM :+1:

Just a small comment below

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Pull request merged**